templater.sh

```diff
@@ -285,13 +298,18 @@ if [[ -f "$template_path" ]]; then
 elif [[ -d "$template_path" ]]; then
   {
     shopt -s globstar nullglob
-    templates=( "$template_path/"* )
-    len=${#templates[@]}
-    len=$((len-1))
-    for i in $(seq 0 $len); do
-        main "${templates[i]}"
-        if [ $i -lt $len ]; then
-        echo -e "$delimiter"
+    if [ "$recursive" == "true" ]; then
+        templates=$(find "$template_path")
+    else
+        templates="$template_path/"*
```

Comment:
this used to be `templates=( "$template_path/"* )` which made it an array.
Now it's just the first file from the glob. That broken :(